Source/WTF/wtf/Assertions.cpp

```diff
@@ -374,6 +374,18 @@ void WTFCrash()
 }
 #endif // !defined(NDEBUG) || !(OS(DARWIN) || PLATFORM(PLAYSTATION))
 
+#if ENABLE(CONJECTURE_ASSERT)
+int wtfConjectureAssertIsEnabled = 0;
```

Comment:
Is there a reason why it's not a `bool`?

Reply:
This variable needs to be declared in Assertions.h, and Assertions.h need to be able to work with C code as well.

Reply:
We already use `bool` in this file, and include `stdbool.h`.

```
WTF_EXPORT_PRIVATE bool WTFWillLogWithLevel(WTFLogChannel*, WTFLogLevel);
```